programmers/서울에서-김서방-찾기.js

```diff
@@ -0,0 +1,7 @@
+function solution(seoul) { 
+    for (let i=0; i<seoul.length; i++){
+        if(seoul[i] === 'Kim'){
+            return '김서방은 '+ i +'에 있다'
```

Comment:
```suggestion
            return `김서방은 ${i}에 있다`
```
이렇게 `` 백틱을 이용하면 ${변수}를 이용해서 한번에 작성이 가능합니다.